src/storage/mapping/ExtensionBasedMapper.ts

```diff
@@ -1,18 +1,27 @@
 import { promises as fsPromises } from 'fs';
 import * as mime from 'mime-types';
 import type { ResourceIdentifier } from '../../ldp/representation/ResourceIdentifier';
-import { TEXT_TURTLE } from '../../util/ContentTypes';
+import { APPLICATION_TRIG, TEXT_TURTLE } from '../../util/ContentTypes';
 import { NotImplementedHttpError } from '../../util/errors/NotImplementedHttpError';
 import { joinFilePath, getExtension } from '../../util/PathUtil';
 import { BaseFileIdentifierMapper } from './BaseFileIdentifierMapper';
 import type { FileIdentifierMapperFactory, ResourceLink } from './FileIdentifierMapper';
 
 export class ExtensionBasedMapper extends BaseFileIdentifierMapper {
   private readonly types: Record<string, any>;
+  private readonly mimeFallback: Record<string, string>;
 
-  public constructor(base: string, rootFilepath: string, overrideTypes = { acl: TEXT_TURTLE, meta: TEXT_TURTLE }) {
+  public constructor(
+    base: string,
+    rootFilepath: string,
+    overrideTypes: Record<string, string> = { acl: TEXT_TURTLE, meta: TEXT_TURTLE, trig: APPLICATION_TRIG },
```

Comment:
Let's drop the default here—or leave it as it was.

The rationale is the following.

This class is expected to perform a "perfect" mapping from extensions to MIME types, with the (sole) exception of Solid-specific MIME types. So `acl` and `meta` are special in the sense that they do not exist outside of the Solid world. In contract, `trig` exists outside of Solid; it's basically a "bug" in `mime-types` (can we PR?).

Reply:
According to the docs it uses the db here and they accept PRs https://github.com/jshttp/mime-db

`application/trig` is a content-type they recognize as existing, but it does not have an official extension associated with it in the db is the issue, as it does not have any in the official databases this library makes use of.

Reply:
I maybe wonder whether we want to rename `overrideTypes` into `customTypes` perhaps.

Reply:
https://github.com/jshttp/mime-db/pull/238